dhis-2/dhis-api/src/main/java/org/hisp/dhis/organisationunit/OrganisationUnit.java

```diff
@@ -557,6 +557,7 @@ public void updateParent(OrganisationUnit newParent) {
    * @return true if this org unit is a descendant of the ancestors.
    */
   public boolean isDescendant(Collection<OrganisationUnit> ancestors) {
+    // TODO: MAS optimize to use Set of UIDs from UserDetails as input instead
```

Comment:
just flagging in case this was not purposely left as a todo (in case you intended to be part of this PR). If intentional then fine.